autofs-5.0.7 - fix incorrect check in flag_is_owned()

From: Ian Kent <raven@themaw.net>

The flag file code isn't used any more but this is clearly incorrect
so fix it in case it gets used sometime in the future.
---
 daemon/flag.c |    7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/daemon/flag.c b/daemon/flag.c
index f8fe163..db9a4bd 100644
--- a/daemon/flag.c
+++ b/daemon/flag.c
@@ -66,12 +66,11 @@ static int flag_is_owned(int fd)
 
 			continue;
 		}
-
-		/* Stale flagfile */
-		if (!tries)
-			return 0;
 	}
 
+	/* Stale flagfile */
+	if (!tries)
+		return 0;
 
 	if (pid) {
 		int ret;
